src/sentry/sentry_metrics/multiprocess.py

```diff
@@ -37,6 +38,27 @@
 DEFAULT_QUEUED_MAX_MESSAGE_KBYTES = 50000
 DEFAULT_QUEUED_MIN_MESSAGES = 100000
 
+SHARED_METRIC_NAMES = {
+    "c:sessions/session@none",
+    "s:sessions/error@none",
+    "s:sessions/user@none",
+    "d:sessions/duration@second",
+}
+SHARED_TAG_STRINGS = {
+    "abnormal",
+    "crashed",
+    "environment",
+    "errored",
+    "exited",
+    "healthy",
+    "init",
+    "production",
+    "release",
+    "session.status",
+    "staging",
+}
+SHARED_STRINGS = {*SHARED_METRIC_NAMES, *SHARED_TAG_STRINGS}
```

Comment:
I am not sure I follow the logic here.
The idea was to statically hardcode a mapping between string and integers. That's why we started the sequence from 16k. 
With this implementation you are just replacing the org id with 0 and relying on the DB to assign an id to these strings. Am I missing something ?

Also please move these in their own module with a large scary comment that says that once a string is in this list cannot be removed or the product breaks.